spec/models/child_site_spec.rb

```diff
@@ -5,6 +5,8 @@
 RSpec.describe ChildSite, type: :model do
   it { should belong_to(:child) }
   it { should belong_to(:site) }
+
+  # TODO: validate started_care, ended_care
```

Comment:
I don't think this will be required for now, so the only validations I can think of are date validations